### PR TITLE
DEVX-6497 - Add Experience Composer to the OpenTok .NET Server SDK

### PR DESCRIPTION
**Pull request by Tr00d**

### CODE REVIEW

**Comment by HunteRoi** [comment]
yo, i need this please merge @jeffswartz

**Review by jeffswartz — COMMENTED** [review]
Should this target the dev branch (not main)?

File: OpenTok/Render/ScreenResolutionConverter.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using EnumsNET;
+using Newtonsoft.Json;
+
+namespace OpenTokSDK.Render
+{
+    /// <summary>
+    ///     Custom converter for serializing ScreenResolution.
+    /// </summary>
+    public class ScreenResolutionConverter : JsonConverter<ScreenResolution>
```
Comment (jeffswartz) [inline OpenTok/Render/ScreenResolutionConverter.cs:10]: Does this class need to be public?

**Review by jeffswartz — CHANGES_REQUESTED** [review]
I added a commit with some docs edits and left some comments.

**Comment by Tr00d** [comment]
The new PR can be found [here](https://github.com/opentok/Opentok-.NET-SDK/pull/210).
I'll close this one after I addressed all comments

**Pull request closed without merging**